src/main/java/com/celarli/commons/vfs/provider/google/GCSFileName.java

```diff
@@ -0,0 +1,40 @@
+package com.celarli.commons.vfs.provider.google;
+
+import org.apache.commons.vfs2.FileName;
+import org.apache.commons.vfs2.FileType;
+import org.apache.commons.vfs2.provider.AbstractFileName;
+
+
+public class GCSFileName extends AbstractFileName {
```

Comment:
The casing of this should be `GcsFileName` to follow how Java code convention deals with acronyms.

Please rename the file & class accordingly.

Reply:
Cancel that, I see the convention in this repo is to fully capitalize GCS.  Ignore these comments please, I will remove them.